packages/botonic-plugin-directus/src/cms/contents.ts

```diff
@@ -0,0 +1,50 @@
+export interface CommonFields {
+  id: string
+  name: string
+  shortText?: string
+  keywords?: string[]
+  followup?: Text | Image
+  customFields?: Record<string, unknown>
+}
+
+export abstract class Content {
+  readonly common: CommonFields
+  constructor(common: CommonFields) {
+    this.common = common
+  }
+  get name(): string {
+    return this.common.name
+  }
+
+  get id(): string {
+    return this.common.id
+  }
+}
+
+export class Text extends Content {
+  readonly text: string
+  readonly buttons?: Button[]
+  constructor(opt: { common: CommonFields; text: string; buttons?: Button[] }) {
+    super(opt.common)
+    this.text = opt.text
+    this.buttons = opt.buttons
+  }
+}
+
+export class Button extends Content {
+  readonly text: string
+  readonly target: string
```

Comment:
`target` should be fixed to a string or should be a union of types like `Text` or `Image` or `string`? 
I'm asking this because I don't know what value will be placed here if the button is targeting a `Text` or an `Image`.